src/pudl/metadata/fields.py

```diff
@@ -1687,6 +1731,10 @@
         "type": "integer",
         "description": "Four-digit year that applies to a particular forecasted value.",
     },
+    "form_type": {
+        "type": "string",
+        "description": "Specific version of SEC 10k filed.",
+    },
```

Comment:
Above we have `exhibit_21_version` which seems analogous, but there we say `version` and here we say `type` -- if both are versions, and this one is specific to the SEC 10K, it seems like it would be more consistent to use `version` in both cases.  So maybe `exhibit21_version` and `sec10k_version`

In our naming conventions we also use the `_type` suffix to indicate "a human readable category from a well defined list of values"

Reply:
That makes sense, fixed.